org.eclipse.lsp4j/src/main/java/org/eclipse/lsp4j/Protocol.xtend

```diff
@@ -444,6 +446,7 @@ class ClientCapabilities {
     /**
      * Experimental client capabilities.
      */
+    @JsonAdapter(JsonElementTypeAdapter.Factory)
     Object experimental
```

Comment:
Why not use `JsonElement`? As far as I understood implementors need to downcast it always?

Reply:
Because in this way implementors are still free to put arbitrary objects into those fields. If we would change the field type to JsonElement, implementors would have to take care of the serialization of their data (of course you are still free to put a JsonElement into the field).

Reply:
ok